internal/api/v1/application/update.go

```diff
@@ -59,6 +60,8 @@ func (hc Controller) Update(c *gin.Context) apierror.APIErrors { // nolint:gocyc
 		return apierror.NewBadRequestError(err.Error())
 	}
 
+	log.Info("updating app", "namespace", namespace, "app", appName, "request", updateRequest)
```

Comment:
the purpose of  `log.Info` is to print out a log (like `echo`in Bash or `console.log` in JS)?

Reply:
Yes. I added quite a bit more to better see what was happening. The output is seen in the kube log of the epinio server pod. --- `k logs -c epinio-server -f :epinio-server`

Reply:
nvm. It seems so. I knew about `fmt` and wondered about the differences. lt seems log is primarily used for logging messages to a log file or standard error stream, while fmt is used for general-purpose formatting and printing.